src/javascripts/_pre-copy-paste.js

```diff
@@ -0,0 +1,101 @@
+mumuki.load(function () {
+
+  var language = {
+    es: {
+      copy: 'Copiar',
+      copied: 'Copiado'
+    },
+    en: {
+      copy: 'Copy',
+      copied: 'Copied'
+    },
+    pt: {
+      copy: 'Copiar',
+      copied: 'Copiado'
+    },
+  }
+
+  function getCopiedText() {
+    var lang = (mumuki.locale || navigator.language || navigator.userLanguage || 'en').split('-')[0];
+    return language[lang];
+  }
+
+  function copyToClipboard(text) {
+    var $temp = $("<textarea>");
+    $("body").append($temp);
+    $temp.val(text).select();
+    document.execCommand("copy");
+    $temp.remove();
+  }
+
+  function getEditor() {
+    return mumuki.page && mumuki.page.editors && mumuki.page.editors[0] && mumuki.page.editors[0]
```

Comment:
Last condition is repeated twice

Reply:
![For more pleasure](http://memecrunch.com/meme/BLM4U/para-mas-placer/image.jpg)